scripts/init.lua

```diff
@@ -13,8 +13,8 @@ end
 
 -- Utility Script for helper functions etc.
 ScriptHost:LoadScript("scripts/utils.lua")
+ScriptHost:LoadScript("scripts/act.lua")
```

Comment:
this shouldn't be needed as ``archipelago.lua`` already loads it with ``require("act")``